src/main/resources/application.yml

```diff
@@ -58,4 +58,5 @@ rabbitmq:
 
 customisation:
   logging:
-    logFlagType: "logAllMessages"
\ No newline at end of file
+    logFlagType:
+      logAllMessages: "yes"
```

Comment:
Valid boolean values are as follows: https://yaml.org/refcard.html
This should likely be `logAllMessages: yes`